migrations/sqls/20240605164904-dispute-categories-up.sql

```diff
@@ -0,0 +1,28 @@
+/* Replace with your SQL commands */
+CREATE TABLE dispute_categories (
+  id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
+  name text NOT NULL,
+  created_at timestamp with time zone NOT NULL DEFAULT now(),
+  updated_at timestamp with time zone NOT NULL DEFAULT now()
+);
+
+INSERT INTO
```

Comment:
issue like perv one you should open 1 PR when changes are bounded together

Reply:
Merged the previous pr.
The change to be resolved:
https://github.com/socious-io/socious-api/pull/580#discussion_r1632595647